Sources/Mantis/Protocols/CropViewControllerDelegate.swift

```diff
@@ -24,6 +24,12 @@ public protocol CropViewControllerDelegate: AnyObject {
     func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController, transformation: Transformation)
     
     func cropViewController(_ cropViewController: CropViewController, didBecomeResettable resettable: Bool)
+    
+    func cropViewControllerDidEnableUndo(_ enable: Bool)
+    
+    func cropViewControllerDidEnableRedo(_ enable: Bool)
+    
+    func cropViewControllerDidEnableReset(_ enable: Bool)
 }
```

Comment:
I think these newly added function names can be more accurate like below
* `cropViewControllerDidUpdateEnableStateForUndo`
* `cropViewControllerDidUpdateEnableStateForRedo`
* `cropViewControllerDidUpdateEnableStateForReset`

Reply:
Changes made.